packages/core/src/form-field/FormFieldLabel.tsx

```diff
@@ -1,20 +1,22 @@
 import { clsx } from "clsx";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
 import { useFormFieldProps } from "../form-field-context";
 import { Label, TextProps } from "../text";
 import { capitalize, makePrefixer } from "../utils";
 
-import { useWindow } from "@salt-ds/window";
-import { useComponentCssInjection } from "@salt-ds/styles";
-
 import formFieldLabelCss from "./FormFieldLabel.css";
 
 const withBaseName = makePrefixer("saltFormFieldLabel");
 
 export const FormFieldLabel = ({
   className,
   children,
+  pronounced = false,
```

Comment:
Should pronounced be a boolean? Has design thought about if there would be an opposite to this

Reply:
@bhoppers2008